docs/getting-started/intro.md

```diff
@@ -10,7 +10,7 @@ Capital pools that automatically generate yield based on opportunities present i
 
 ## Governance
 
-The Yearn ecosystem is controlled by YFI token holders who submit and vote on off-chain proposals that govern the ecosystem. Proposals that generate a majority support \(&gt;50% of the vote\) are implemented by a 9 member multi-signature wallet. Changes must be signed by 6 out of the 9 wallet signers in order to be implemented. The members of the multi-signature wallet were voted in by YFI holders and are subject to change from future governance votes. Please refer to our FAQ for [the list of the multisig signers](https://docs.yearn.finance/resources/faq#who-is-on-the-multisig) and more information about the [governance process](https://docs.yearn.finance/resources/faq#governance).
+The Yearn ecosystem is controlled by YFI token holders who submit and vote on off-chain proposals that govern the ecosystem. Proposals that generate a majority support \(&gt;50% of the vote\) are implemented by a 9 member multi-signature wallet. Changes must be signed by 6 out of the 9 wallet signers in order to be implemented. The members of the multi-signature wallet were voted in by YFI holders and are subject to change from future governance votes. Please refer to the [multisig documentation](../security/multisig) for the list of multisig signers. For more info about the governance process, please consult the [Governance FAQ](../resources/faq#governance) and [YIP-61: Governance 2.0](https://gov.yearn.finance/t/yip-61-governance-2-0/10460).
```

Comment:
please don't use relative links like this: `../security/multisig` and `../resources/faq#governance` . We've been using full links in the whole docs